packages/react-tabster/src/hooks/useFocusIndicatorStyle.ts

```diff
@@ -30,18 +31,17 @@ const getFocusOutlineStyles = (options: FocusOutlineStyleOptions) => {
   const outlineOffsetRight = (outlineOffset as FocusOutlineOffset)?.right || outlineOffset;
 
   return {
-    borderColor: 'transparent',
+    ...shorthands.borderColor('transparent'),
     ':after': {
       content: '""',
       position: 'absolute',
       pointerEvents: 'none',
-      boxSizing: 'outline-box',
```

Comment:
`outline-box` is not supported value

https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing